### DISCUSSION TITLE
Will Radix continue to be maintained?

### DISCUSSION THREAD

**Discussion by manspaniel** [radix-ui/primitives#2305] [Help]
We've used Radix a lot over the last couple of years, and I do love it!

We also still use Stitches as part of our stack, and have a lot of love for it as well.

With Stitches no longer being maintained (we're moving to Panda CSS going forward), it does raise the question for us — is Radix likely to follow suite at some point? Are there plans to maintain it long-term?

(Apologies if this has been addressed somewhere else!)

**Comment by zxti** [comment]
With all due respect to the great work that the Radix team invested, I can recommend React Aria Components (I'm unaffiliated).

- Similar API, I easily migrated from Radix
- Many more components, e.g. date range picker
- More flexible, can drop down to hooks
- Better performance
- Fewer dependencies
- Actively maintained and further developed
- Backed by Adobe engineering team, Devon Govett, et al

**Comment by benoitgrelard** [comment]
Stitches is not maintained anymore, but Radix UI (Primitives, Themes, Colors, etc) definitely are. They are the foundation of our very own [design system](https://workds.workos.dev/) at WorkOS.
